### DISCUSSION TITLE
ReboundHub.dll blocked by Huorong antivirus software

### DISCUSSION THREAD

**Discussion by KeeLee2005** [IviriusCommunity/Rebound#105] [General]
Huorong Antivirus has reported the virus Trojan/MSIL.Runner.d with ID 7EEE9D56CE472629

At first, the UI opens after installation and displays Rebound 11 is not installed (Screenshot 1). But after installing it here, the software seems to crash and won't open anyway.

Screenshots:
![QQ_1730127932595](https://github.com/user-attachments/assets/92f39cce-6f0e-4ea2-95b9-15ca0184b845)
![QQ_1730127962563](https://github.com/user-attachments/assets/dbe75c02-69e7-4e62-b088-351172886f57)

Exported log:
[Rebound hub.txt](https://github.com/user-attachments/files/17544557/Rebound.hub.txt)

Windows Version: Windows 11 24H2 Build 26100.2161 Release Preview Channel
Rebound Hub version: ALPHA v0.0.3

**Comment by IviriusMain** [comment]
Weird, might have to contact the company to get Rebound Hub whitelisted

**Comment by IviriusMain** [comment]
This *should* be fixed now
